app/model/repository/ShadowAssignmentPointsRepository.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Model\Repository;
+
+use App\Model\Entity\ShadowAssignment;
+use App\Model\Entity\ShadowAssignmentPoints;
+use App\Model\Entity\User;
+use Kdyby\Doctrine\EntityManager;
+
+/**
+ * @method ShadowAssignmentPoints findOrThrow($id)
+ */
+class ShadowAssignmentPointsRepository extends BaseRepository {
+
+  public function __construct(EntityManager $em) {
+    parent::__construct($em, ShadowAssignmentPoints::class);
+  }
+
+  /**
+   * Find best solutions of given assignments for user.
+   * @param ShadowAssignment[] $shadowAssignments
+   * @param User $user
+   * @return ShadowAssignmentPoints[]
+   */
+  public function findEvaluationsForAssignment(array $shadowAssignments, User $user): array {
+    return $this->findBy([
+      "author" => $user,
+      "shadowAssignment" => $shadowAssignments,
```

Comment:
Sorry, I do not see, how this should work. Why is $shadowAssignments an array, passed to ManyToOne property shadowAssignment? I was expecting just one shadow assignment.

Reply:
Effing shiet... no this will not work

Reply:
Well... actually yes... this will work, if you pass array to `shadowAssignment`, it'll find you all points entities for user and given shadow assignments...